src/components/__test__/IdeaItem.test.js

```diff
@@ -0,0 +1,51 @@
+import IdeaAdder from '../IdeaItem.vue';
+import { shallowMount, createLocalVue } from '@vue/test-utils';
+import Buefy from 'buefy';
+import mock from '../../test/mockBack';
+
+describe('IdeaAdder', () => {
+  let localVue;
+  beforeEach(() => {
+    const Vue = createLocalVue();
+    Vue.use(Buefy);
+    localVue = Vue;
+  });
+  test('render component', () => {
+    const wrapper = shallowMount(IdeaAdder, {
+      localVue,
+      propsData: {
+        ...mock,
```

Comment:
this is how the props are "copied " i guess ?

Reply:
Yes this is the desestructuring in js, some like :
```
mock is an object
...mock = {
                  name : some,
                  start : number 
                  ...
}